bims/views/contributions.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+from django.contrib.auth.models import User
```

Comment:
instead of this import, please use settings.AUTH_USER_MODEL
because BiologicalCollectionRecord using this setting as foreign key